app/shared/actions/blockchains.js

```diff
@@ -6,10 +6,11 @@ import { setSettings, setSettingWithValidation } from './settings';
 import { clearActionsCache } from './accounts';
 import { clearProducerInfo } from './producers'
 
-function swapBlockchain(chainId) {
+function swapBlockchain(chainId, account, authorization) {
   return (dispatch: () => void, getState) => {
     const { blockchains } = getState();
     const blockchain = find(blockchains, { chainId });
+    dispatch({ type: types.SYSTEM_DUPLICATING_ACCOUNT_PENDING, payload: { account, authorization } });
```

Comment:
So this triggers every time anyone switches blockchains? 

I don't think this belongs here 🤔